src/st/class-page-builder-integration.php

```diff
@@ -5,6 +5,8 @@
  */
 class WPML_Page_Builders_Integration {
 
+	const UPDATE_TRANSLATED_POST_PRIORITY = 10;
```

Comment:
@adrianosferreira shouldn't the constant reflect the name of the action isntead of the callback?

Reply:
@sciamannikoo makes sense, fixed.